tests/e2e/e2e_setup_test.go

```diff
@@ -368,14 +412,20 @@ func (s *IntegrationTestSuite) configureChain(chainId string, validatorConfigs [
 	// If upgrade is skipped, we can use the chain initialization logic from
 	// current branch directly. As a result, there is no need to run this
 	// via Docker.
+	if !s.isFork {
+		s.forkHeight = 0
+	}
```

Comment:
`s.forkHeight` should be 0 by default and set to something else if fork is enabled when parsing environment variables
```suggestion
```